### PR TITLE
Adding original author notes

### PR DESCRIPTION
**Pull request by sgill2**
Added original author credit to example/example.py and blues/ncmc_switching.py.

### CODE REVIEW

**Comment by davidlmobley** [comment]
@sgill2 - somehow I did not see this when it came in. Can you tag me when things are ready for review to make sure I don't miss them? (And nag me if I don't respond within a couple days!) Reviewing now.

**Comment by sgill2** [comment]
Added an acknowledgements section to README.md as suggested above.

File: README.md [hunk 1]
```diff
@@ -87,3 +87,5 @@ An example of how to set up a simulation sampling the binding modes of toluene b
 
 ## Implementing other non-equilibrium moves
 An optional argument for SimNCMC.runSim() , movekey allows users to implement their own moves into the simulation workflow. To add your own moves make a subclass of SimNCMC. The custom method can use self.nc_context or self.nc_integrator to access the NCMC context or NCMC integrator in the runSim() method. Then, insert the method into the movekey. As an example, say you implemented a smart darting move in a method called smartdart(). Then for the keymove argument use [[smartdart, [range]]], where range is a list of integers that specify when you want to apply the move. In general the keymove argument takes a list of lists. The first item in the list references the function, while the second references a list which specifies what steps you want to apply the function.
+##Acknowledgements
+We would like to thank Patrick Grinaway and John Chodera for their basic code framework for NCMC in OpenMM, and John Chodera and Christopher Bayly for their helpful discussions.
```
Comment (davidlmobley) [inline README.md]: Can we reference a specific repo or repos for the code framework?

**Comment by davidlmobley** [comment]
@sgill2 - can you wrap this one up?

**Comment by sgill2** [comment]
Oh sorry, somehow I thought I already merged this PR.

**Comment by davidlmobley** [comment]
Thanks. I don't like having pull requests hanging unless they have "[WIP]" in the title. ;)

**Pull request merged**